cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -623,19 +623,23 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
           {{- range $i, $e := .Frontend.Backends}}
           {
             name: '{{upper .Name}}_BASE_URL'
-            value: 'https://{{.Name}}.${containerAppsEnvironment.outputs.defaultDomain}'
+            value: 'https://${ {{bicepName .Name}}.outputs.name}.${containerAppsEnvironment.outputs.defaultDomain}'
```

Comment:
If a frontend app A has a call dependency on backend app B, refer to the output of container app Bicep module, to make them topologized.